server/js/dao.js

```diff
@@ -612,6 +612,23 @@ exports.newBot = async function(mapId, botNftId, xp, name, walletId, ownerEntity
   }
 }
 
+exports.getShopInventory = async function(shopId) {
+  const options = {
+    headers: {
+      'X-Api-Key': API_KEY,
+      'Content-Type': 'application/json'
+    }
+  }
+  const url = `${LOOPWORMS_LOOPERLANDS_BASE_URL}/shopInventory.php?shopName=${shopId}`;
+  const shopResponse = await axios.get(url, options);
+  try {
+    return shopResponse.data;
+  } catch (error) {
```

Comment:
can you log the error? I know it may reduce the performance but its probably worth it for insight.

Reply:
sure..I'll add a log there